bitcrowd/bitstyles
# [#710] Update color palette

Fixes #710 

## Changes

- Updates color design tokens to the new colors, taken from the styleguide in zeplin
- Color variations now use the naming format `light-4`, `light-3`, `light-2`, `light-1`, `base`, `dark-1`, `dark-2`, `dark-3`, `dark-4`, instead of `lightest`, `lighter`, `light`, `base`, `dark`, `darker`, `darkest`. Note that there’re new points on the scale — `dark-4` and `light-4` did not exist in the old scale.
- Updates color usage throughout the codebase

## How to QA

Pull this branch, and run `yarn storybook`.

Most changes in this PR are updating the use of `u-fg`, `u-bg`, and `u-border` utility classes. The core changes are in `settings/_color-palette.scss`.

Check the color-palette page in storybook, and click around some components.

⚠️ the components may not look beautiful — updating them to properly use these colors will come in future PRs, when we update the base styles and component styles to match the new designs (see zeplin).

## 📸

![design tokens color palette page](https://user-images.githubusercontent.com/2479422/219383732-3ca5a471-0a75-47f6-a719-15116d9f10a4.png)

## Checks

Delete if not applicable:
